pkg/addons/default/assets/aws-node.yaml

```diff
@@ -81,7 +81,7 @@ spec:
       tolerations:
         - operator: Exists
       containers:
-        - image: 602401143452.dkr.ecr.us-west-2.amazonaws.com/amazon-k8s-cni:v1.5.0
+        - image: docker.io/weaveworks/amazon-k8s-cni:v1.6.0-rc1-2-g813f2829
```

Comment:
This should be `602401143452.dkr.ecr.us-west-2.amazonaws.com/amazon-k8s-cni:v1.5.4`, just like in [config/v1.5/aws-k8s-cni.yaml](https://github.com/aws/amazon-vpc-cni-k8s/blob/release-1.5.4/config/v1.5/aws-k8s-cni.yaml#L84)